packages/evolution-frontend/src/components/admin/validations/InterviewMap.tsx

```
@@ -434,72 +580,74 @@ const InterviewMap: React.FunctionComponent<InterviewMapProps> = (props: Intervi
+                        >
+                            <FontAwesomeIcon icon={faLayerGroup} />
+                        </button>
+
+                        {/* Professional layer options - shown when button is clicked */}
+                        {showLayerSelect && (
+                            <div className="admin__map-layer-dropdown">
+                                {/* OSM Option */}
+                                <div
+                                    onClick={() => handleLayerChange('osm')}
+                                    className={`admin__map-layer-option ${!showAerialTiles ? 'active' : ''}`}
+                                >
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)